en/docs/references/performance-analysis.md

```diff
@@ -1,18 +1,77 @@
+- You can avoid missing significant performance characteristics when combining multiple applications, including external applications.
 - You can correctly assess the cost and feasibility of running the application in a production environment.
+- You can reduce the time and cost spent on debugging performance issues by eliminating or minimizing iterative load testing efforts.
+
+## Use the Performance Analyzer
+
+You can use the Choreo Performance Analyzer on the Web Editor in low-code mode. 
+
+### Analyze the performance-critical path in real-time
+
+To view the real-time analysis of the performance-critical path of your component, follow the steps below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** to access the Web Editor.
+
+#### Overall performance of the performance-critical path
+
+On the low-code editor, you will see a bar with the summarized overall performance estimates for each resource function. By default, this banner displays the forecasted latency and throughput for a selected range of user count for the performance-critical path. The upper bound of the user count range is set based on the availability of historical data. Since the Performance Analyzer tool derives these metrics in real-time,  any changes to the component will update the metrics in real time. 
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click  **Reveal performance-critical path** on the top banner to highlight it in green in the low-code diagram.
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that you have selected.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component. 
```

Comment:
'By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green.' This part has been repeated above.

Reply:
Yes it is done intentionally, as there can be users who skim through the doc without following it end to end. Since this section focuses on a sub topic, and because the starting point is to reveal the graph, mentioned it here.

Reply:
Yet repeating the same words twice is not good for a reader who goes through the whole document.